Sources/ForageSDK/Component/ForagePANTextField/ForagePANTextField.swift

```diff
@@ -10,7 +10,7 @@ public enum CardType: String {
     case EBT = "ebt"
 }
 
-public class ForagePANTextField: UIView, Identifiable, ForageElement, ForageElementDelegate {
+public class ForagePANTextField: UIView, Identifiable, ForageElement, CardInformation, ForageElementDelegate {
```

Comment:
> thoughts on introducing e a `derivedCardInfo` member rather than conforming `ForagePANTextField`to`CardInformation`?

This would be my vote as well because I think it organizes these fields in a logical and extensible way 👍 

As far as naming goes:
* `cardMetaInfo`
* `cardMeta`

I'm not too opinionated here